src/solc_0.8/polygon/child/land/PolygonLand.sol

```diff
@@ -9,8 +9,16 @@ import "./PolygonLandBaseToken.sol";
 contract PolygonLand is PolygonLandBaseToken {
     address public polygonLandTunnel;
 
-    constructor() {
+    bool internal _initialized;
+
+    modifier initializer() {
+        require(!_initialized, "ERC721BaseToken: Contract already initialized");
+        _;
+    }
+
+    function initialize() external initializer {
```

Comment:
should the method init be versionned ? 
check PolygonAssetV2, initialize calls initV2

To be discussed